src/cpr_sdk/ssm.py

```diff
@@ -0,0 +1,34 @@
+import boto3
+
+from cpr_sdk.search_adaptors import VespaSearchAdapter
+
+
+def get_aws_ssm_param(param_name: str, region_name: str = "eu-west-1") -> str:
+    """Retrieve a parameter from AWS SSM"""
+    ssm = boto3.client("ssm", region_name=region_name)
+    response = ssm.get_parameter(Name=param_name, WithDecryption=True)
+    return response["Parameter"]["Value"]
```

Comment:
I wonder if this will ever be used outside of a prefect flow, in which case perhaps we could start a prefect submodule in the sdk that could later have other things, like a deployments wrapper, etc. What do you think?